snaps/features/custom-evm-accounts/index.md

```diff
@@ -273,17 +274,89 @@ It resolves the request using
 and the Snap resolves the request using [`snap_manageAccounts`](../../reference/snaps-api.md#snap_manageaccounts),
 notifying MetaMask of the result.
 
-## Supported signing methods
+## EOA methods
```

Comment:
SEO seems to favor [externally owned accounts](https://www.semrush.com/analytics/keywordoverview/?q=externally%20owned%20accounts&db=us&date=202402&device=0)

Might be useful to have that as the heading